rustfmt.toml

```diff
@@ -1,2 +1,2 @@
 write_mode = "Overwrite"
-
+closure_block_indent_threshold = 1
```

Comment:
I personally prefer block indent, so I'm fine with this change, and I also think the official rust style is moving towards preferring block indent too.

However, I expect the reason is that without this, rustfmt seems to be buggy and formats some stuff horribly, particularly in unit.rs. eg
````
             constants::DW_FORM_addr => {
                 return parse_address(input, unit.address_size()).map(|(rest, addr)| {
-                    let attr = Attribute {
+                                                                         let attr = Attribute {
                         name: spec.name(),
                         value: AttributeValue::Addr(addr),
                     };
-                    (rest, attr)
-                });
+                                                                         (rest, attr)
+                                                                     });
             }
````